articles/tokens/add-custom-claims.md

```diff
@@ -0,0 +1,34 @@
+---
+description: How to add custom claims to tokens using Rules
+topics:
+  - tokens
+  - access-tokens
+  - id-tokens
+  - custom-claims
+  - scopes
+  - claims
+  - namespaces
+contentType:
+  - how-to
+useCase:
+  - invoke-api
+---
+
+# Add Custom Claims to Tokens
+
+You can add custom claims to your [Access Tokens](/tokens/overview-access-tokens) or [ID Tokens](/tokens/id-token)) using [Rules](/rules). The claim name must conform to a namespaced format to avoid possible collisions with standard OIDC claims. Otherwise, it is not possible to add arbitrary claims to ID Tokens or Access Tokens.
+
+The format you should follow is:  `http://my-namespace/claim-name`.
+
```

Comment:
> You can add custom claims to your [Access Tokens](/tokens/overview-access-tokens) or [ID Tokens](/tokens/id-token)) using [Rules](/rules).
> 
> ## Requirements
> 
> The name of your custom claim must conform to the namespaced format (shown below) to avoid possible collisions with standard OIDC claims: ~Otherwise, it is not possible to add arbitrary claims to ID Tokens or Access Tokens.~
> 
> `http://my-namespace/claim-name`.

Reply:
Done